src/lpython/semantics/python_attribute_eval.h

```diff
@@ -67,7 +67,9 @@ struct AttributeHandler {
 
     ASR::asr_t* get_attribute(ASR::expr_t *e, std::string attr_name,
             Allocator &al, const Location &loc, Vec<ASR::expr_t*> &args, diag::Diagnostics &diag) {
-        ASR::ttype_t *type = ASRUtils::expr_type(e);
+        ASR::ttype_t *type = ASRUtils::type_to_str(ASRUtils::expr_type(e)) == "list const"
```

Comment:
I think using `ASRUtils::type_to_str(ASRUtils::expr_type(e)) == "list const"` is not the correct way. Instead you should be checking for `ASR::is_a<ASR::Const_t>()` if you want to know if the type is `const`.

Reply:
I had started with the method you have stated, but it threw an error every time. I tried checking the type to see if it was a `Const`. When it failed, I tried checking if the contained type was a list; that failed too. Please see https://github.com/lcompilers/lpython/pull/2567#discussion_r1515412371

Reply:
I am pushing the changes. Please see https://github.com/lcompilers/lpython/pull/2567#discussion_r1515443211